components/browser/menu/src/main/java/mozilla/components/browser/menu/item/BrowserMenuImageText.kt

```diff
@@ -23,6 +24,19 @@ internal fun ImageView.setTintResource(@ColorRes tintColorResource: Int) {
     }
 }
 
+internal fun TextView.tintedDrawable(@DrawableRes imageResource: Int, @ColorRes tintColorResource: Int): Drawable {
+    val draw = context.getDrawable(imageResource)!!
+
+    if (tintColorResource != NO_ID) {
+        DrawableCompat.setTint(
```

Comment:
nit: could just use `draw.setTint` since we're only supporting Lollipop and above.